### DISCUSSION TITLE
Updating the supporting members

### DISCUSSION THREAD

**Discussion by fibonacci1729** [bytecodealliance/SIG-Registries#42] [General]
I'm updating the supporting members in [SIG-Registries Governance](https://github.com/bytecodealliance/governance/blob/main/SIGs/SIG-Registries/proposal.md).

The working list as of now:

- Birch, Johnnie [@jlb6740](https://github.com/jlb6740) (Intel)
- Brown, Kyle [@Kylebrown9](https://github.com/Kylebrown9) (SingleStore)
- Butcher, Matt [@technosophos](https://github.com/technosophos) (Fermyon)
- Hardock, Brian [@fibonacci1729](https://github.com/fibonacci1729) (Fermyon)
- Hayes, Bailey [@ricochet](https://github.com/ricochet) (Cosmonic)
- Huene, Peter [@peterhuene](https://github.com/peterhuene) (Fastly)
- Kulakowski, George [@kulakowski-wasm](https://github.com/kulakowski-wasm) (Fastly)
- Matei, Radu [@radu-matei](https://github.com/radu-matei) (Fermyon)
- Martin, Lann [@lann](https://github.com/lann) (Fermyon)
- McCallum, Nathaniel [@npmccallum](https://github.com/npmccallum) (Profian)
- Squillace, Ralph [@squillace](https://github.com/squillace) (Microsoft)
- Thomas, Taylor [@thomastaylor312](https://github.com/thomastaylor312) (Cosmonic)
- Volosatovs, Roman [@rvolosatovs](https://github.com/rvolosatovs) (Profian)
- Wagner, Luke [@lukewagner](https://github.com/lukewagner) (Fastly)
- Zhou, Jiaxiao [@Mossaka](https://github.com/Mossaka) (Microsoft)

Feel to 👍 or suggest changes/additions/corrections.

Draft PR: https://github.com/bytecodealliance/governance/pull/13/files

**Comment by npmccallum** [comment]


**Reply by fibonacci1729** [reply]
Updated, thanks!

**Comment by fibonacci1729** [comment]
If I am forgetting anyone please comment and i'll update the list. Thanks!

**Comment by fibonacci1729** [comment]
I don't have permissions to merge https://github.com/bytecodealliance/governance/pull/13/files so if someone here has write permissions, please feel free to merge.
